packages/threat-composer/src/data/status/mitigationStatus.json

```diff
@@ -0,0 +1,26 @@
+[
+    {
+        "label": "Identified",
+        "value": "mitigationIdentified",
+        "description": "Mitigation has been identified",
+        "iconName": "suggestions"
+    },
+    {
+        "label": "In-Progress",
+        "value": "mitigationInProgress",
+        "description": "Mitigation is in-progress",
+        "iconName": "status-in-progress"
+    },
+    {
+        "label": "Resolved",
+        "value": "mitigationResolved",
+        "description": "Mitigation has been completed to an agreed scoped",
+        "iconName": "status-positive"
+    },
+    {
+        "label": "Abandoned",
+        "value": "mitigationResolvedAbandoned",
+        "description": "Will not be implementing this mitigation",
+        "iconName": "flag"
+    }
```

Comment:
Upon reflection, 'Abandoned' carries a negative connotation - and that it is simply left behind or neglected. I propose we change to 'Will not action' which carries slightly less negative connotation in my opinion, and is clear that it's a _decision_:

```
    {
        "label": "Will not action",
        "value": "mitigationResolvedWillNotAction",
        "description": "Will not be implementing this mitigation",
        "iconName": "flag"
    }
```